frontend/src/utils/omakase.js

```diff
@@ -0,0 +1,90 @@
+import { fetchAuthSession } from "aws-amplify/auth";
+import {
+  EventBridgeClient,
+  PutEventsCommand,
+} from "@aws-sdk/client-eventbridge";
+
+export async function executeExport(formData, editTimeranges, flows) {
+  const { credentials } = await fetchAuthSession();
+
+  const region =
+    //@ts-ignore
+    import.meta.env.VITE_APP_AWS_REGION;
+
+  if (region === undefined) {
+    console.error("VITE_EXPORT_EVENT_BRIDGE_REGION is not defined in .env");
+  }
+
+  const client = new EventBridgeClient({
+    region: region,
+    credentials: credentials,
+  });
+  let configuration = {};
+
+  if (formData.operation === "Segment Concatenation") {
+    configuration.format = formData.format;
+    configuration.output = {
+      bucket: formData.bucket !== "" ? formData.bucket : undefined,
+      path: formData.path !== "" ? formData.path : undefined,
+      filename: formData.filename !== "" ? formData.filename : undefined,
+    };
+  } else if (formData.operation === "Flow Creation") {
+    configuration.label = formData.label;
+  }
+
+  const operation = formData.operation.replaceAll(" ", "_").toUpperCase();
+
+  const eventBusName =
+    //@ts-ignore
+    import.meta.env.VITE_APP_OMAKASE_EXPORT_EVENT_BUS ?? "omakase-tams";
```

Comment:
I would like us to be consistent here and add this to our constants file and then use that here instead.